pkg/apis/kops/v1alpha1/zz_generated.conversion.go

```diff
@@ -2125,6 +2125,7 @@ func autoConvert_v1alpha1_KubeProxyConfig_To_kops_KubeProxyConfig(in *KubeProxyC
 	out.HostnameOverride = in.HostnameOverride
 	out.Master = in.Master
 	out.Enabled = in.Enabled
+	out.ProxyMode = in.ProxyMode
```

Comment:
For some reason we are missing this ... we will need to get this into the 1.9 release as well.  I am pushing a pr.

Reply:
Ok great, cheers 👍